src/Microsoft.Extensions.SecurityHelper.Sources/SecurityHelper.cs

```diff
@@ -17,7 +17,8 @@ internal static class SecurityHelper
         /// Merges a new claims principal, placing all new identities first, and eliminating
         /// any empty unauthenticated identities from context.User
         /// </summary>
-        /// <param name="identity"></param>
+        /// <param name="existingPrincipal">The ClaimsPrincipal containing existing ClaimsIdentities.</param>
```

Comment:
Can the typenames be changed to be type `<see ...>` elements or whatever that thing is?